src/features/commands/help/help-command.service.ts

```diff
@@ -0,0 +1,23 @@
+import _ from "lodash";
+import { Message } from "discord.js";
+import { HelpsEnum } from "../../../enums/helps.enum";
+import { DisplayMessageService } from "../../../services/display-message/display-message.service";
+
+export class HelpCommandService {
+  private static _instance: HelpCommandService;
+
+  public static getInstance(): HelpCommandService {
+    if (_.isNil(HelpCommandService._instance)) {
+      HelpCommandService._instance = new HelpCommandService();
+    }
+    return HelpCommandService._instance;
+  }
+
+  public message(message: Message, targetHelp: HelpsEnum): Promise<Message> {
```

Comment:
Do not really like the typo here for `targetHelp`

Reply:
Is that "target" that is bothering you? °°

Reply:
@asmodee-bellanie yeah I think that this is a bad variable name.
For once because it should be helpTarget and secondly because helpTarget is not helping me to understand the purpose of that variable.
But this is your code so if you are comfortable with it just keep it.

Reply:
Ok. I'll see what I can do to replace it.